modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc

```diff
@@ -515,7 +513,7 @@ Clients wishing to sync previously removed documents must use the resetCheckpoin
 :param-tags: autopurge-override
 include::{root-partials}block_tabbed_code_example.adoc[]
 :param-tags!:
-<.> Here we have opted to turn off the auto purge behavior. By default this is ON (true)
+<.> Here we have opted to turn off the auto purge behavior. By default auto purge is enabled (false)
```

Comment:
I think we should remove (false) as the value will depend on the platform. In C, the value will be true and the others will be false.